packages/cli/src/lib/redis/api.ts

```diff
@@ -71,11 +71,20 @@ export default (app: string, database: string | undefined, json: boolean, heroku
 
   return {
     request<T>(path: string, method: HttpVerb = 'GET', body = {}) {
-      const headers = {Accept: 'application/json'}
+      const headers = {Accept: 'application/vnd.heroku+json; version=3'}
```

Comment:
Is it necessary to modify this header? I know that's the specific header API expects, does data-api care?